react-common/components/theming/ThemeCard.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from "react";
+import { ThemePreview } from "./ThemePreview";
+import { Card } from "../controls/Card";
+
+interface ThemeCardProps {
+    theme: pxt.ColorThemeInfo;
+    onClick?: (theme: pxt.ColorThemeInfo) => void;
+}
+
+export class ThemeCard extends React.Component<ThemeCardProps> {
```

Comment:
Just curious, why did you choose to make this a React component?

Reply:
Was just on autopilot, didn't really realize there was an alternative.

After some searching, am I correct in understanding that since it doesn't use state, it can just be something like `const ThemeCard = (props: ThemeCardProps) => { ... }`?

Reply:
Wow, I already did it in my other components and didn't realize 😅 Will change this one to match.